siliconcompiler/dependencyschema.py

```diff
@@ -1,9 +1,12 @@
 import os.path
 
-from .schema.baseschema import BaseSchema
-from .schema.editableschema import EditableSchema
-from .schema.parameter import Parameter, Scope
-from .schema.namedschema import NamedSchema
+from siliconcompiler.schema.baseschema import BaseSchema
+from siliconcompiler.schema.editableschema import EditableSchema
+from siliconcompiler.schema.parameter import Parameter, Scope
```

Comment:
Is Dependency the right place for package schema information? Seems like it might be.  What is the up to date diagram showing which schema go in which class?